rules/prefer-starts-ends-with.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const doesNotContain = (string, chars) => chars.every(char => string.indexOf(char) === -1);
```

Comment:
You can use `String#includes` here instead of indexOf. It's supported in Node.js 4.